Gems/Multiplayer/Code/Include/Multiplayer/Components/LocalPredictionPlayerInputComponent.h

```diff
@@ -90,23 +99,32 @@ namespace Multiplayer
         // Anti-cheat accumulator for clients who purposely mess with their clock rate
         NetworkInputArray m_lastInputReceived;
 
-        AZ::ScheduledEvent m_autonomousUpdateEvent; // Drives autonomous input collection
+#if AZ_TRAIT_SERVER
```

Comment:
Not terribly important, but maybe coalesce the client and server member sections just to minimize the amount of #if / #endif 's we have in the header..